packages/cli/package.json

```diff
@@ -39,9 +39,11 @@
     "sinon": "^4.5.0",
     "yeoman-assert": "^3.1.1",
     "yeoman-environment": "^2.0.6",
-    "yeoman-test": "^1.7.0"
+    "yeoman-test": "^1.7.0",
+    "typescript": "^3.0.1"
```

Comment:
I am curious, what do we need `typescript` for? Why is it a direct dev-dependency?

Reply:
I am not  sure if it is my machine only. I ran npm install `@phenomnomnominal/tsquery --save` only, and it complained about the `typescript` missing. They just specify this dependency for dev only. But in runtime, they still need it. The only way to get rid of this error was to place the `typescript` dependency in our CLI package.json .

Reply:
Also, I found in their tests they import some constants also from typescript, especially to identify the type of node etc.

Reply:
@bajtos , they answered my issue and will fix it today. They will apply typscript as a `peerDependcy`  rather than the current scenario `devDependency`.